recipes/mykrobe/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -x
 
-export CFLAGS="-I$PREFIX/include"
+export CFLAGS="-I$PREFIX/include -fPIC"
```

Comment:
I've tried this locally and it still gave the same error. I'm going a little crazy staring at c compilation errors 😅

Reply:
The error indicates its using htslib.a - can it be that you try to compile statically and not dynamically?

Reply:
Not quite sure. The htslib compiling is done by mccortex as it is one of it's dependencies. It's strange because there is a [bioconda recipe for mccortex](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/mccortex) already and I seem to have copied what they do.  
Admittedly, the mccortex required by mykrobe here is a fork that is ever so slightly modified, but it hasn't diverged that much from the original mccortex.  
FWIW, if I require `gcc <5.0` instead of the `{{ compiler('c') }}` require it builds and tests ok locally.